dts/arm/silabs/xg24/bgm240sa22vna.dtsi

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2023 Fr. Sauter AG
+ * Copyright (c) 2025 Silicon Laboratories Inc.
+ * Copyright (c) 2025 Ephraim Westenberger
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <mem.h>
+#include <silabs/xg24/efr32mg24.dtsi>
```

Comment:
Sorry for not noticing earlier: This should include `efr32bg24.dtsi` instead. While there isn't any difference in Zephyr right now, once 802.15.4 support is added for mg24, this will be wrong.